commcare_export/env.py

```diff
@@ -222,9 +222,17 @@ def emitted_tables(self):
 
 @unwrap('val')
 def str2bool(val):
+    if val is None:
+        return False
+
     if isinstance(val, bool):
         return val
-    return val and str(val).lower() in {'true', 't', '1'}
+
+    if not isinstance(val, six.string_types):
+        val = str(val)
```

Comment:
I guess it somewhat depends on whether we want to always coerce to the desired type, even if it means losing some information, or if we want to raise errors for invalid input. If the former, then `val.decode('utf-8', 'replace')` should always be safe. If the latter, then maybe `val.decode('utf-8')` is better? Choosing which one probably depends on context.